src/components/Button/index.js

```diff
@@ -15,9 +16,10 @@ const Button = ({ theme, href, children, onClick }) => {
 
   if (href) {
     return (
-      <a className={className} href={href}>
-        {children}
-      </a>
+      <NextLink href={href}>
+        {/* eslint-disable-next-line */}
```

Comment:
What are you disabling??? xD

Reply:
`a11y` - no href warning.

Reply:
You can go to the `eslintrc` and change this rule: 
```
    "jsx-a11y/anchor-is-valid": [
      "error",
      {
        "components": [
          "Link"
        ],
        "specialLink": [
          "to"
        ]
      }
    ],
```

To include `NextLink` in that components list

Reply:
I'll change this to disable that specific rule

Reply:
Updated